.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,51 @@
+name: Lint and Test Charts
+
+on:
+  push:
+    branches-ignore:
+    - 'main'
+    - 'repository'
+    
+  pull_request:
+    branches:
+    - '**'
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-20.04
```

Comment:
minor nit, but `ubuntu-latest` means this doesn't need to be bumped every few years :slightly_smiling_face:

Reply:
Sure. I'll modify it.

I had pinned it due to the lag when `latest` still resulted in Ubuntu 18.04 LTS even though Ubuntu 20.04 LTS was released. It was a bit frustrating since `latest` didn't actually mean latest.

Reply:
I personally like to use specific number as latest can change and some code does not work the same way between OS versions sometimes. This one tho, I am not too worried to use latest as it is just a lint test.